src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -273,7 +273,7 @@
     <value>Excluded apps</value>
   </data>
   <data name="FancyZones_ExcludeApps_TextBoxControl.Header" xml:space="preserve">
-    <value>To exclude an application from snapping to zones add its name here (one per line). Excluded apps will react to the Windows Snap regardless of all other settings.</value>
+    <value>To exclude an application from snapping to zones add its name here (one per line). These apps will react only to Windows Snap.</value>

```

Comment:
nit, add a comma: To exclude an application from snapping to zones, add its name here (one per line).